docs/UserGuide.md

```diff
@@ -5,18 +5,18 @@
 * [Features](#features)
   * [Notes about the command format](#notes-about-the-command-format)
   * [Viewing the command list: `menu`](#viewing-the-command-list-menu)
-  * [List all events: `list`](#list-all-events-list)
-  * [Add an event, participant or item: `add`](#add-an-event-participant-or-item-add)
-  * [Remove an event or participant: `remove`](#remove-an-event-or-participant-remove-)
-  * [View all participants or items for an event: `view`](#view-all-participants-or-items-for-an-event-view)
-  * [Edit the information of an event or a participant: `edit`](#edit-the-information-of-an-event-or-a-participant-edit)
-  * [Mark an event as done: `mark`](#mark-an-event-as-done-mark)
-  * [Marks a participant as present: `mark`](#marks-a-participant-as-present-mark)
-  * [Marks an item as accounted for: `mark`](#marks-an-item-as-accounted-for-mark)
-  * [Copies participant list: `copy`](#copies-participant-list-copy)
-  * [Sorts event list: `sort`](#sorts-event-list-sort)
-  * [Filters event list: `filter`](#filters-event-list-filter)
-  * [Find participant: `find`](#find-participant-find)
+  * [Listing all events: `list`](#listing-all-events-list)
+  * [Adding an event, participant or item: `add`](#adding-an-event-participant-or-item-add)
+  * [Removing an event or participant: `remove`](#removing-an-event-or-participant-remove-)
+  * [Viewing all participants or items for an event: `view`](#viewing-all-participants-or-items-for-an-event-view)
+  * [Editing the information of an event or a participant: `edit`](#editing-the-information-of-an-event-or-a-participant-edit)
+  * [Marking an event as done: `mark`](#marking-an-event-as-done-mark)
+  * [Marking a participant as present: `mark`](#marking-a-participant-as-present-mark)
+  * [Marking an item as accounted for: `mark`](#marking-an-item-as-accounted-for-mark)
```

Comment:
if for add, remove and edit we combine all three to one line, we should do the same for this,

Reply:
I've combined the three sections into one section. You can review it if it flows well or not.